token-syncer/integration/token_syncer/basic_test.clj

```diff
@@ -37,9 +44,14 @@
 (deftest ^:integration test-environment
   (log/info "Running: test-environment")
   (testing "presence of environment variables"
-    (log/info "env.WAITER_URIS" (System/getenv "WAITER_URIS"))
-    (is (System/getenv "WAITER_URIS"))
-    (is (> (count (waiter-urls)) 1))))
+    (testing "waiter cluster uris"
+      (log/info "env.WAITER_URIS" (System/getenv "WAITER_URIS"))
+      (is (System/getenv "WAITER_URIS"))
+      (is (> (count (waiter-urls)) 1)))
+
+    (testing "kitchen command"
+      (log/info "env.WAITER_TEST_KITCHEN_CMD" (System/getenv "WAITER_TEST_KITCHEN_CMD"))
+      (is (not (str/blank? (System/getenv "WAITER_TEST_KITCHEN_CMD")))))))
```

Comment:
I don't understand what these tests are accomplishing

Reply:
These assertions are ensuring the integration tests are running with the required environment variables being present.

Reply:
A comment explaining this would be nice